sympy/solvers/ode.py

```diff
@@ -5784,7 +5787,7 @@ def _solve_undetermined_coefficients(eq, func, order, match):
     return Eq(f(x), gsol.rhs + psol)
 
 
-def _undetermined_coefficients_match(expr, x):
+def _undetermined_coefficients_match(expr, x, eq_homogenous=S.Zero):
```

Comment:
It should be homogeneous rather than homogenous

Reply:
Do we need to have a default value for this function?

Reply:
> Do we need to have a default value for this function?

in my opinion it should have default value because when user wants to know only `family of given function` so this was achieved by importing `_undetermined_coefficients_match(expr,x)` so if the default argument is not present then user has to explicitly pass `0` as third parameter.